lib/src/main/java/com/squareup/cycler/RecyclerMutations.kt

```diff
@@ -72,6 +72,12 @@ class MutationExtensionSpec<T : Any> : ExtensionSpec<T> {
    */
   var dragAndDropEnabled = false
 
+  /**
+   * Determine whether items can be dragged left and right in the recycler view or not.
+   * If [dragAndDropEnabled] == false, this flag is ignored.
```

Comment:
@helios175 how do you feel about this logic? You have to first enable DnD and only then does left/right enabled kick in.